components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
```

Comment:
Por que hubo que agregar esto? Yo lo borré y corrió bien.

Reply:
@nykka resulta que los componentes que van a ser utilizados en el lado del cliente y que utilizan hooks como `useState` y `useEffect` deben utilizar la directiva `"use client"`. 
De lo contrario al ser importados en el directorio `app` arrojan error.

[Client components](https://beta.nextjs.org/docs/rendering/server-and-client-components#client-components)

Reply:
Como dice Manuel, quise importar el componente en la pagina home para hacer pruebas y me daba un error de hidratacion que decia que usara "use client"